### DISCUSSION TITLE
关于“为系统增加内置环境变量”的讨论，征求社区意见

### DISCUSSION THREAD

**Discussion by holdno** [holdno/gopherCron#21] [Ideas]
期望能够以任务粒度实现环境变量的注入  
该环境变量可应用于：  
- 任务命令的配置  
- 任务进程内环境变量的注入  

讨论内容：  
- 环境变量的注入姿势  
- 应该/期望被内置(默认存在)的环境变量，例如命令中使用 GC_ENV_DAY 可以获取当天的日期 2023-05-11

**Comment by holdno** [comment]
- workflow每次手动执行时，弹窗配置环境变量，配置完成后再点击确定执行workflow

**Comment by chenbt-hz** [comment]
支持添加环境变量模板，通过模版 + 任务tag 批量完成环境变量注入

**Reply by holdno** [reply]
可以给一些更详细的描述吗？ 通过这个描述我有点想不出具体落地的姿势是怎样的 😃

**Reply by chenbt-hz** [reply]
一点粗浅的想法。具体细节等有空再学习借鉴一下这个项目。

**Reply by linw1995** [reply]
模板可以这么弄，环境变量其实就是 Key Value 对，那表单就好做了。再支持几个控件，左边填环境变量名，右边选择控件。后面在执行的时候，就可以通过控件配置参数。控件前期支持日期，日期时间，文本输入应该就够了
